en/docs/setup/deployment-guide.md

```diff
@@ -234,7 +235,56 @@ WSO2 supports the following membership schemes for clustering
             a range only for the last portion of the IP address. You should also keep in mind that the smaller 
             the range, the faster the time it takes to discover members since each node has to scan a lesser 
             number of potential members. 
-        
+            
+        ??? tip "Click to see the instructions for AWS ECS membership scheme"  
+            !!! note
+                AWS ECS membership scheme is only supported from WUM-424342 onwards.  
+                      
+            1. First, you need a working AWS ECS Cluster.
+            for instructions on creating a cluster. Please take note of the following things when creating the cluster.
+                -   Note down the `name` and `VPC CIDR block` of the cluster as we will require them for later configurations.
```

Comment:
1. **Create a working AWS ECS Cluster. For instructions, see [add link]. Note the following when creating a cluster.**

- Note the `name` and `VPC CIDR block` of the cluster as **you** will require them **later for configurations**. 
- Ensure that the `Container instance IAM role` that you assign to the ECS cluster **has** permission to **access?** the following policy.

Reply:
Resolved in https://github.com/wso2/docs-is/pull/895/commits/23568508a359cfdc535bfcdedd1782cf6b3fb7e4